src/App.js

```diff
@@ -14,11 +15,33 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state={
```

Comment:
У тебя установлен линтер?
```suggestion
  state = {
```

Reply:
Установлен, но перестал работать. Обновил vscode, сейчас вроде бы ОК. Хотя на данную ошибку он почему-то все равно не реагирует.